dss-core/src/main/java/io/github/ztkmkoo/dss/core/exception/handler/DssRestExceptionHandler.java

```diff
@@ -0,0 +1,70 @@
+package io.github.ztkmkoo.dss.core.exception.handler;
+
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceResponse;
+import io.github.ztkmkoo.dss.core.actor.rest.service.DssRestActorService;
+import io.github.ztkmkoo.dss.core.exception.annotation.ExceptionHandler;
+import io.github.ztkmkoo.dss.core.exception.annotation.ServiceExceptionHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class DssRestExceptionHandler {
```

Comment:
Hi,
The name of the class(`DssRestExceptionHandler`) looks the implementation of DssExceptionHandler interface.

How bout change name to DssRestExceptionHanlderResolver or else?

Reply:
I agree with that. The DssRestExceptionHandlerResolver is good name.

Naming is the hardest thing in the world. 👍